packages/devtools_app/lib/src/screens/debugger/evaluate.dart

```diff
@@ -176,6 +237,11 @@ class _ExpressionEvalFieldState extends State<ExpressionEvalField>
                   ),
                 );
               },
+              // Disable ligatures, so the suggestions of the auto complete work correcly.
```

Comment:
What happens with ligatures enabled?

Reply:
This is what happens:

https://user-images.githubusercontent.com/1659532/166835194-ca3bef40-74be-4143-a39c-9d8c019ca49b.mov

T‌he ligature in this example is the `fi` of in the word `profile`. I'm not quite sure why this behaviour happens though, maybe do you have an idea?

T‌o build the content within the `TextField`, we use this code:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/1659532/166835563-802afeff-81e7-4065-860b-d2f9e1c802bf.png">

We have two `TextSpan`, the first displays the text `prof` (which is the actual typed text). The other `TextSpan` will display `ile`, which is the suggestion. The only reasonable cause I can come up with is that something goes wrong because the ligature is split between two different `TextSpan`.

I tried placing a "Zero Width Non-Joiner" (https://unicode-explorer.com/c/200C) character to prevent this, but this will break the caret position.

That's why the easiest solution to this is to disable ligatures for the text in the TextField.

Reply:
Oh weird! Thanks for the explanation. Fine to keep as is with ligatures disabled